src/ralph/cli.py

```diff
@@ -165,6 +166,26 @@ def extract(parser):
         click.echo(event)
 
 
+@cli.command()
+@click.option(
+    "-f",
+    "--format",
+    "format_",
+    type=click.Choice(["edx"]),
+    required=True,
+    help="Input events format to validate",
+)
+def genson(format_):
+    """Generates JSON schemas from input events that are not covered by validate."""
```

Comment:
I think we don't need to generate schemas we already have ? But the actual reason is because this helped to remove the `simple server event`s from the scope as they can't be separated by `event_source`/`event_type`.

Reply:
> I think we don't need to generate schemas we already have?

It will constitute a validation test of the proposed approach :wink:

Reply:
Good point) You are right) I will update this part to include all events)